src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { PersonLink } from '../components/PersonLink';
+
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader/Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
it's not an issue just a suggestion. you can simplify it:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```